server/views/partials/bespokeConditionsSummary.njk

```diff
@@ -1,32 +1,29 @@
 {% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
 
-{%  macro bespokeConditions(licence, isEditable, title, hideHeading, bespokeConditionsToDisplay) %}
-    {% if hideHeading != true %}
-        <div class="govuk-grid-row check-answers-header">
-            <div class="govuk-grid-column-three-quarters">
-                <h2 class="govuk-heading-m">{{ title or "Bespoke licence conditions" }}</h2>
-            </div>
-        </div>
-    {% endif %}
-    {% set bespokeConditions = [] %}
+{% macro bespokeConditions(licence, isEditable, hideHeading, showConditionCountSelectedText, bespokeConditionsToDisplay) %}
```

Comment:
would it be better to move these to an object structure args so you can see what the keys are when calling these?

Reply:
Adjusted in latest commit